cmd/tfimport/README.md

```diff
@@ -15,32 +15,38 @@ generated by the Terrarom Engine.
 
 ## Usage
 
+### Installing and Running
+
+To install and run (recommended):
+
 ```shell
-go install ./cmd/tfimport
-tfimport --input_dir ~/path/to/terraform/configs/dir/
+go install github.com/GoogleCloudPlatform/healthcare-data-protection-suite/cmd/tfimport
```

Comment:
To be clearer I still think we should show a dummy cd to show that we should be in the tf dir or put a comment.

Also, does `github.com/GoogleCloudPlatform/healthcare-data-protection-suite/cmd/tfimport` fetch the release at HEAD?

Reply:
It does (technically fetches the default branch, "master" in our case).